app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1 +1,31 @@
 /* Placeholder for backend dummy app */
+SpreePayuIntegration = {
+  updateSaveAndContinueVisibility: function() {
+    if (this.isButtonHidden()) {
+      $(this).trigger('hideSaveAndContinue')
+    } else {
+      $(this).trigger('showSaveAndContinue')
+    }
+  },
+  isButtonHidden: function () {
+    paymentMethod = this.checkedPaymentMethod();
+    return (SpreePayuIntegration.paymentMethodID && paymentMethod.val() == SpreePayuIntegration.paymentMethodID);
+  },
+  checkedPaymentMethod: function() {
+    return $('div[data-hook="checkout_payment_step"] input[type="radio"][name="order[payments_attributes][][payment_method_id]"]:checked');
+  },
+  hideSaveAndContinue: function() {
+    $("#checkout_form_payment [data-hook=buttons]").hide();
+  },
+  showSaveAndContinue: function() {
+    $("#checkout_form_payment [data-hook=buttons]").show();
+  }
+}
+
+$(document).ready(function() {
+  SpreePayuIntegration.updateSaveAndContinueVisibility();
+  paymentMethods = $('div[data-hook="checkout_payment_step"] input[type="radio"]').click(function (e) {
```

Comment:
I'm a bit scary that this gem interacts so deeply with spree store. What if the store would have totally custom template without those hooks? I believe it should be at least very well documented and mentioned why this is how it is.